pages/dkp/kaptain/1.3.0/release-notes/1.3.0/index.md

```diff
@@ -70,3 +70,41 @@ In DKP 2.1, Kaptain's dependencies are managed through Kommander, the centralize
 * CUDA 11.2
 * MXNet 1.9
 
+## Known issues
+
+### cert-manager workaround for Kaptain
+
+Due to an oversight, some Kommander versions do not properly handle certificate renewal for the Cluster CA and certificates that are created for Kommander applications, which also affects Kaptain. While the effects can vary, the most common failure is the inability to launch Kaptain notebooks in Jupyter.
+
+#### Regenerate the secrets in DKP
+
+A permanent fix for the issue requires upgrading to Kommander 2.2.1 or higher. If you are running other versions of DKP, refer to the cert-manager expiration workaround documentation for DKP [2.1.0](../../../../kommander/2.1/release-notes/2.1.0#cert-manager-expiration-workaround), [2.1.1](../../../../kommander/2.1/release-notes/2.1.1#cert-manager-expiration-workaround) or [2.1.2](../../../../kommander/2.1/release-notes/2.1.2#cert-manager-expiration-workaround) to run a docker container that extends the validity of the Cluster CA to 10 years and fixes the certificate reload issue.
+
+Once this is done, you can fix the issue on Kaptain’s side.
+
+#### Regain access to Kaptain
+
+This gives you back the capability of launching notebooks in Jupyter:
+
+1.  Kaptain has one certificate that you have to delete to force a refresh, and one that you can update manually for Istio:
+
+    ```bash
+    kubectl delete secrets kubeflow-gateway-certs -n kubeflow --force
+    ```
+
+1.  Obtain the CA from one of the other recreated certs:
+
+    ```bash
+    kubectl get secret kommander-traefik-certificate -o jsonpath='{.data.ca\.crt}' > ca.crt
```

Comment:
```suggestion
    kubectl get secret kommander-traefik-certificate -n kommander -o jsonpath='{.data.ca\.crt}' > ca.crt
```
in kaptain 1.3 we poupulate the `kubeflow-oidc-ca-bundle` secret the same way we do for kaptain 2.0 (using `kommander-traefik-certificate`) however this command is missing the namespace value.
([link](https://github.com/mesosphere/kaptain/blob/bb63818cab015fab224d26808a97e35c76d0563c/hack/helm-installer/scripts/kaptain-installer.sh#L21) to how the helm installer would fetch this value for kaptain 1.3)